Libs/MRML/Core/vtkMRMLColorNode.cxx

```diff
@@ -375,6 +375,23 @@ const char *vtkMRMLColorNode::GetColorName(int ind)
 }
 
 //---------------------------------------------------------------------------
+int vtkMRMLColorNode::GetColorIndexByName(const char *name)
+{
+  if (this->GetNamesInitialised() && name != NULL)
```

Comment:
Shouldn't we just initialize color names if they have not been initialized yet? 

For example, as it is done in vtkMRMLColorNode::GetColorName(int ind):

  if (!this->GetNamesInitialised())
    {
    this->SetNamesFromColors();
    }